maxim/logger/groq/helpers.py

```
@@ -0,0 +1,164 @@
+    capabilities.
+
+    All methods are static and can be called directly on the class without
+    instantiation.
+    """
+
+    @staticmethod
+    def sync_stream_helper(
+        stream: Stream[ChatCompletionChunk],
+        generation: Generation,
+        trace: Optional[Trace],
+        is_local_trace: bool,
```

Ruff (0.11.9)

Boolean-typed positional argument in function definition

(FBT001)